docs/reference/hooks.rst

```diff
@@ -976,3 +975,51 @@ Document serving
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
   Called when Wagtail is about to serve a document. The callable passed into the hook will receive the document object and the request object. If the callable returns an ``HttpResponse``, that response will be returned immediately to the user, instead of serving the document. Note that this hook will be skipped if the :ref:`WAGTAILDOCS_SERVE_METHOD <wagtaildocs_serve_method>` setting is set to ``direct``.
+
+
+Snippets
+--------
+
+Hooks for working with registered Snippets.
+
+.. _after_edit_snippet:
+
+``after_edit_snippet``
+~~~~~~~~~~~~~~~~~~~~~~
+
+  Called when a Snippet is edited. The callable passed into the hook will receive the model instance, the request object. If the callable returns an ``HttpResponse``, that response will be returned immediately to the user, and Wagtail will not proceed to call ``redirect()`` to the listing view.
+
+  .. code-block:: python
+
+    from wagtail.core import hooks
+
+    @hooks.register('after_edit_snippet')
+    def after_snippet_update(request, instance):
+        # your code here
+        # print(instance.pk)
```

Comment:
Might be good to add something a bit more specific here as per other docs examples for hooks
e.g.
```
from django.http import HttpResponse
...
return HttpResponse(f"Congrats on editing a snippet with id {instance.pk}", content_type="text/plain")
```